internal/integration/mysql_test.go

```diff
@@ -36,8 +36,12 @@ var myTests struct {
 func myRun(t *testing.T, fn func(*myTest)) {
 	myTests.Do(func() {
 		myTests.drivers = make(map[string]*myTest)
-		for version, port := range map[string]int{"56": 3306, "57": 3307, "8": 3308, "Maria107": 4306, "Maria102": 4307, "Maria103": 4308} {
-			db, err := sql.Open("mysql", fmt.Sprintf("root:pass@tcp(localhost:%d)/test?parseTime=True", port))
+		for version, port := range map[string]int{"TiDB": 4309} {
```

Comment:
Don't forget to include the other drivers again

Reply:
Whoops, thx